sadcowpng/sharedspaces/tests.py

```diff
@@ -1557,75 +1557,74 @@ def test_space_close(self):
+                                       "space_wifi": True,
+                                       "space_restrooms": False,
+                                       "space_food_drink": True}
+        TestCase.test_space_form = CreateSpaceForm(data=TestCase.default_space_data)
+        TestCase.test_space_form.is_valid()
+        name = TestCase.test_space_form.cleaned_data['space_name']
+        description = TestCase.test_space_form.cleaned_data['space_description']
+        max_capacity = TestCase.test_space_form.cleaned_data['space_max_capacity']
+        space_address1 = TestCase.test_space_form.cleaned_data['space_address1']
+        space_address2 = TestCase.test_space_form.cleaned_data['space_address2']
+        space_zip_code = TestCase.test_space_form.cleaned_data['space_zip_code']
+        space_city = TestCase.test_space_form.cleaned_data['space_city']
+        space_state = TestCase.test_space_form.cleaned_data['space_state']
+        space_country = TestCase.test_space_form.cleaned_data['space_country']
+        noise_level_allowed = int(TestCase.test_space_form.cleaned_data["space_noise_level_allowed"][0])
+        noise_level = int(TestCase.test_space_form.cleaned_data["space_noise_level"][0])
+        wifi = TestCase.test_space_form.cleaned_data['space_wifi']
+        restroom = TestCase.test_space_form.cleaned_data['space_restrooms']
+        food_drink = TestCase.test_space_form.cleaned_data['space_food_drink']
+        space_open = TestCase.test_space_form.cleaned_data['space_open']
+
+        # pulls data from form and fills out model fields to save space in table
+        test_space = Space(space_name=name,
+                           space_description=description,
+                           space_max_capacity=max_capacity,
+                           space_address1=space_address1, space_address2=space_address2,
+                           space_zip_code=space_zip_code,
+                           space_city=space_city, space_state=space_state, space_country=space_country,
+                           space_noise_level_allowed=noise_level_allowed, space_noise_level=noise_level,
+                           space_wifi=wifi,
+                           space_restrooms=restroom, space_food_drink=food_drink,
+                           space_open=space_open)
 
+        test_space.save()
+
+        # now create and save the space data model
+        space_date = TestCase.test_form_date.cleaned_data['date']
+        space_start_time = TestCase.test_form_date.cleaned_data['time_start']
+        space_end_time = TestCase.test_form_date.cleaned_data['time_end']
+        space_id = test_space
+        date_time = SpaceDateTime(space_date=space_date,
+                                  space_start_time=space_start_time,
+                                  space_end_time=space_end_time,
+                                  space_id=space_id)
+
+        date_time.save()
```

Comment:
seems like things got shifted. Please make sure nothing was overwritten though. Doesn't seem like it but just to make sure.